spec/models/account_spec.rb

```diff
@@ -711,6 +711,14 @@
     it 'does not match URL query string' do
       expect(subject.match('https://example.com/?x=@alice')).to be_nil
     end
+
+    it 'matches usernames immediately following the letter ß' do
+      expect(subject.match('Hello toß @alice from me')[1]).to eq 'alice'
+    end
+
+    it 'matches usernames containing uppercase characters' do
+      expect(subject.match('Hello to @aLice@Example.com from me')[1]).to eq 'aLice@Example.com'
+    end
```

Comment:
I added this test to make sure that the case insensitive matching is working correctly. This test passes on `main`.

Reply:
I really like this idea. Would you mind adding the same for hashtags?

Reply:
Done ✅

Reply:
Thanks a lot!